autopilot/autopilot.go

```diff
@@ -177,11 +177,20 @@ func (ap *Autopilot) configHandlerPUT(jc jape.Context) {
 	}
 }
 
+func (ap *Autopilot) renterKeyHandlerGET(jc jape.Context) {
+	var hk consensus.PublicKey
+	if jc.DecodeParam("hostkey", &hk) != nil {
+		return
+	}
+	jc.Encode(ap.deriveRenterKey(hk))
+}
```

Comment:
I don't think we need to do this. Key derivation should be moved from the autopilot into the worker, I just haven't gotten around to it.

Reply:
ok removed it